provision/provisioner.sh

```diff
@@ -54,3 +54,5 @@ apt install -y postfix
 
 # netdata
 bash <(curl -Ss https://my-netdata.io/kickstart.sh) all --non-interactive
+
+tar -xf /var/lib/haconiwa/images/ssh.image.tar -C /var/lib/haconiwa/rootfs/nginx-10-0-5-10/
```

Comment:
The extract of image is done at the initial setup of the container in dispatcher. What does this mean?

https://github.com/FastContainer/nginx-haconiwa/pull/16/files#diff-92007c9f8d9ac747304ec558f0648086L133